--- a/bin/aclocal.in
+++ b/bin/aclocal.in
<<<<<<< HEAD
@@ -370,6 +370,12 @@ sub scan_m4_dirs ($$@)
=======
@@ -371,6 +371,12 @@ sub scan_m4_dirs ($$@)
>>>>>>> 94392b39ec (稳定版本发布)
 
   foreach my $m4dir (@dirlist)
     {
+      if (! -d $m4dir)
+        {
+          msg ('override', "warning: skipping not existing directory `$m4dir'");
+          next;
+        }
+
       if (! opendir (DIR, $m4dir))
 	{
 	  # TODO: maybe avoid complaining only if errno == ENONENT?
